roles/openshift_aws/tasks/dns_route53_record.yml

```diff
@@ -0,0 +1,25 @@
+---
+- debug: msg="{{ l_openshift_dns_records }}"
+
+- name: querying elb
+  ec2_elb_facts:
+    names: "{{ l_openshift_dns_records['value'] }}"
+    region: "{{ openshift_aws_region }}"
+  register: elb_facts
+  when:
+    - "'elb' in l_openshift_dns_records['kind']"
+    - "'CNAME' in l_openshift_dns_records['type']"
+
+- name: creating record
+  route53:
+    command: create
+    overwrite: no
+    private_zone: "{{ (l_openshift_dns_records['scheme'] == 'internal') | ternary('yes','no') }}"
```

Comment:
we should name this key "private_zone" instead of 'scheme', and just set yes or no directly, instead of doing this.